plugins/editor/CMakeLists.txt

```diff
@@ -137,9 +140,14 @@ else()
     target_include_directories(sfizz_editor PRIVATE ${sfizz-gio_INCLUDE_DIRS})
     target_link_libraries(sfizz_editor PRIVATE ${sfizz-gio_LIBRARIES})
 endif()
-target_link_libraries(sfizz_editor PRIVATE sfizz::colorspaces sfizz::stb_image
-    sfizz::bit_array sfizz::filesystem sfizz::pugixml)
-
+target_link_libraries(sfizz_editor PRIVATE
+    sfizz::colorspaces
+    sfizz::stb_image
+    sfizz::bit_array
+    sfizz::filesystem
+    sfizz::pugixml
+    plugins::git_build_id
```

Comment:
It feels strange here. I'm guessing you had some issues. Would switching `plugins::git_build_id` from PRIVATE to PUBLIC here allow to remove the hard-coded path in `target_include_directories` above?